### DISCUSSION TITLE
[Cloud Engineer] Eliminate creation and maintenance of unused assets

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/patterns#32] [Ideas]
https://docs.aws.amazon.com/wellarchitected/latest/sustainability-pillar/eliminate-creation-and-maintenance-of-unused-assets.html

Intent: Analyze application assets (such as pre-compiled reports, data sets, and static images) and asset access patterns to identify redundancy, underutilization, and potential decommission targets. Consolidate generated assets with redundant content (for example, monthly reports with overlapping or common data sets and outputs) to eliminate the resources consumed when duplicating outputs. Decommission unused assets (for example, images of products that are no longer sold) to free consumed resources and reduce the number of resources used to support the workload.

**Comment by holanita** [comment]
[Issue created](https://github.com/Green-Software-Foundation/green-software-patterns/issues/68)
